src/Core/AdminConsole/Enums/EventSystemUser.cs

```diff
@@ -6,4 +6,5 @@ public enum EventSystemUser : byte
     SCIM = 1,
     DomainVerification = 2,
     PublicApi = 3,
+    TwoFactorDisabling = 4,
```

Comment:
I'm not sure about this name, I welcome suggestions

Reply:
Maybe `TwoFactorDisabled`? `Disabled` sounds more like a `state`, but I don't feel strongly about this. The name you choose isn't a blocker for me.

Reply:
Makes sense to me, I'll rename it